app/controllers/network_router_controller.rb

```diff
@@ -12,4 +12,259 @@ class NetworkRouterController < ApplicationController
   def self.display_methods
     %w(instances cloud_subnets)
   end
+
+  def button
```

Comment:
could you make a method per button action out of this?

we are trying to cleanup the button and x_button methods and this introduces more of what we need to get rid of

see e.g. https://github.com/ManageIQ/manageiq/pull/11977

there should be a method per action and that's it

it's hard to get any plugability for the actions unless we get rid of complex button methods